src/main/java/trackerbot/TrackerBot.java

```diff
@@ -0,0 +1,171 @@
+package trackerbot;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Main class of the Console Bot
+ */
+public class TrackerBot {
+    private FileIO f;
+    private TaskList taskList;
+    private UI ui;
+
+    public enum Commands {
+        MARK,
+        UNMARK,
+        DELETE,
+        ADDTASK,
+        TODO,
+        DEADLINE,
+        EVENT,
+        BYE,
+        LIST,
+        FIND,
+        DEFAULT,
+        INVALID
+    }
+
+    /**
+     * Returns an instance of the console TrackerBot with the correct settings
+     */
+    public TrackerBot() {
+        try {
+            this.f = new FileIO();
+            this.taskList = this.f.readFileContents();
+            this.ui = new UI();
+        } catch (TrackerBotException | FileNotFoundException e) {
+            ConsoleDisplayStyle.printBasicStyling(0,0, e.getMessage());
+        }
+    }
+
+    /**
+     * Starts running the TrackerBot in the console
+     */
+    public void run() {
+
+        this.ui.greet();
+
+        //Bot in Process
+        Scanner inputScanner = new Scanner(System.in);
+        int maxInputLength = 0;
+        boolean exitLoop = false;
+
+        while (!exitLoop && inputScanner.hasNextLine()) {
+            String userInput = inputScanner.nextLine();
+            int inputLength = userInput.length();
+            Trio<Commands, Integer, TaskList> commandsIndexTaskTrio = Parser.parseUserInput(userInput, this.taskList);
+            Commands userCommand = commandsIndexTaskTrio.getHead();
+            Integer taskIndex = commandsIndexTaskTrio.getBody();
+            TaskList taskTargetList = commandsIndexTaskTrio.getTail();
+            Task taskTarget = taskIndex != null ? taskTargetList.getTask(taskIndex) : null;
+
+            //Bot Replies Instead of Echo
+            int stylingIndex = 7;
+            switch (userCommand) {
```

Comment:
To adhere to coding standards, could align case clauses with the switch statement. 
```suggestion
            switch (userCommand) {
            case BYE:
```